### PR TITLE
feat(discover-homepage): Update Discover homepage docs for GA

### PR DESCRIPTION
**Pull request by narsaynorath**
Updating docs to prepare for GA launch

### CODE REVIEW

File: src/docs/product/discover-queries/index.mdx [hunk 1]
```diff
@@ -90,9 +92,7 @@ Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="rea
 
 ### Set as Default
 
-<Include name="early-adopter-note.mdx" />
-
-If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.
+When you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.
 
 You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.
```
Comment (narsaynorath) [inline src/docs/product/discover-queries/index.mdx:97]: I also want to shim in the whole, you can go to a saved query and also set that as your homepage but I'm not exactly sure how to word it.

> You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. You can also find "Set as Default" in [saved queries](#query-results) as well.

**Review by edwardgou-sentry — APPROVED** [review]

File: src/docs/product/discover-queries/index.mdx [hunk 2]
```diff
@@ -21,7 +21,9 @@ This feature is available only if your organization is on either a Business or T
 
 ![Full view of the Discover Homepage with query cards and button to build new queries.](./discover-homepage.png)
 
-The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> page offers views of all your saved and pre-built queries so you can quickly see areas that need immediate attention. For example, if a query on errors shows a spike has occurred in the past 24 hours, you might want to investigate that first. Each query is saved as a card that displays a summarized view of the following data:
+The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage will open the query builder where you can immediately begin building a new query. By default, this will show the pre-built "All Events" query. Columns in the homepage can be [edited and saved, or saved from other queries](#set-as-default). To navigate back to the [query management page](#saved-queries), where pre-built and saved queries can be found, click the "Saved Queries" button.
```
Comment (imatwawana) [inline src/docs/product/discover-queries/index.mdx:24]: A couple things:

- I don't understand what you mean by "saved from other queries" and the section on default homepages doesn't really explain this so I've edited it to mirror the wording we used in that section
- I understand putting the link in the words "query management page" but it's confusing to link it to a heading called Saved Queries so I've moved the link.

```suggestion
The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin building a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set it as your default](#set-as-default). To navigate back to the query management page(#saved-queries), where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.
```

File: src/docs/product/discover-queries/index.mdx (hunk 1 above)
Comment (imatwawana) [inline src/docs/product/discover-queries/index.mdx:97]: Wondering if you can set other pre-built queries as defaults? Should we be mentioning this?
```suggestion
You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on saved queries, which you can find by clicking the "Saved Queries" button.
```

**Review by imatwawana — COMMENTED** [review]
I've made a couple edits and left a couple questions. I'm also thinking we should add something just after the _Pre-built Queries_ and _Saved Queries_ headings to indicate how you access these (using the "Saved Queries" button).

**Comment by narsaynorath** [comment]
@imatwawana so I've made the changes you suggested and I also added a little sentence in the prebuilt and saved query sections mentioning how to navigate to them.

As for your questions r.e. whether pre-built and saved queries can be set as default, they can, so thanks for calling that out in the "Set as Default" section

File: src/docs/product/discover-queries/index.mdx (hunk 2 above)
Comment (imatwawana) [reply src/docs/product/discover-queries/index.mdx]: Let's do it!

File: src/docs/product/discover-queries/index.mdx (hunk 1 above)
Comment (imatwawana) [reply src/docs/product/discover-queries/index.mdx]: Updated my suggestion

Comment (imatwawana) [reply src/docs/product/discover-queries/index.mdx]: Nevermind

File: src/docs/product/discover-queries/index.mdx [hunk 3]
```diff
@@ -21,7 +21,9 @@ This feature is available only if your organization is on either a Business or T
 
 ![Full view of the Discover Homepage with query cards and button to build new queries.](./discover-homepage.png)
 
-The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> page offers views of all your saved and pre-built queries so you can quickly see areas that need immediate attention. For example, if a query on errors shows a spike has occurred in the past 24 hours, you might want to investigate that first. Each query is saved as a card that displays a summarized view of the following data:
+The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin building a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set it as your default](#set-as-default). To navigate back to the query management page, where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.
```
Comment (imatwawana) [inline src/docs/product/discover-queries/index.mdx:24]:
```suggestion
The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin creating a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set the custom version as your default](#set-as-default). To navigate back to the query management page, where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.
```

File: src/docs/product/discover-queries/index.mdx [hunk 4]
```diff
@@ -90,11 +92,9 @@ Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="rea
 
 ### Set as Default
 
-<Include name="early-adopter-note.mdx" />
-
-If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.
+When you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.
 
-You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.
+You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on saved queries, which you can find by clicking the "Saved Queries" button.
```
Comment (imatwawana) [inline src/docs/product/discover-queries/index.mdx:97]:
```suggestion
You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on both saved and pre-built queries, which you can find by clicking the "Saved Queries" button.
```

**Review by imatwawana — APPROVED** [review]
A few minor wording tweaks and I added that bit about pre-built queries being able to be set as default. Otherwise, looks good!

**Pull request merged**